src/actions/gifs.js

```diff
@@ -0,0 +1,7 @@
+const addGif = (num) => {
```

Comment:
viết action nên viết thế này nè em 
```suggestion
export const addGif = (num) => {
```

Reply:
Em đã lại action